app/src/main/java/com/belatrixsf/allstars/ui/contacts/ContactPresenter.java

```diff
@@ -48,19 +52,48 @@ public void setProfileEnabled(boolean profileEnabled) {
         this.profileEnabled = profileEnabled;
     }
 
-    public void getEmployeeList() {
-        view.showProgressIndicator();
-        employeeService.getEmployeeList(new AllStarsCallback<SearchEmployeeResponse>() {
-            @Override
-            public void onSuccess(SearchEmployeeResponse searchEmployeeResponse) {
-                view.showEmployees(searchEmployeeResponse.getEmployeeList());
-            }
+    public void setInActionMode(boolean inActionMode) {
+        this.inActionMode = inActionMode;
+    }
 
-            @Override
-            public void onFailure(ServiceError serviceError) {
-                showError(serviceError.getErrorMessage());
-            }
-        });
+    public boolean isInActionMode() {
+        return inActionMode;
+    }
+
+    public List<Employee> forSavingEmployees(){
+        return employees;
+    }
+
+    public boolean forSavingInActionMode(){
+        return inActionMode;
+    }
+
```

Comment:
the same as 
 +    public boolean isInActionMode() {
 +        return inActionMode;
 +    }